extensions/extension.pb.go

```diff
@@ -1,24 +1,12 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
 // source: extension.proto
 
-/*
```

Comment:
I think this was included by accident. I don't think this should be included. Should I remove it?

Reply:
Let's keep it. It looks like this changed because the protoc-gen-go output changed. I tested and an extension still built with the change, and extension support is still experimental (and incomplete).